scripts/helper_functions.sh

```diff
@@ -14,4 +14,46 @@ function update_version(){
     return 12
   fi
   return 0
-}
\ No newline at end of file
+}
+
+function updateVersionForAllSubprojects() {
```

Comment:
i would prefer `updateAllVersionInformation`, because the example projects are technically not subprojects ( and we use the term subproject differently in the build.gradle file)